tests/testthat/_snaps/count.md

```diff
@@ -3,6 +3,6 @@
     Code
       dt %>% count(name = 10) %>% collect()
     Condition
-      Error in `tally()`:
+      Error in `tally_count()`:
```

Comment:
Unrelated to the group fix in this PR, I think this should say "Error in `check_name()`" to be consistent with dplyr, which means removing the `call = caller_env()` from the `abort()` call.

Reply:
I'd add the commit myself, but all my snapshots are completely off and I'm not sure why. They all say "Error <rlang_error>" instead of "Condition"